code/game/objects/items/charter.dm

```diff
@@ -110,7 +110,7 @@
 
 /obj/item/station_charter/banner/rename_station(designation, uname, ureal_name, ukey)
 	set_station_name(designation)
-	minor_announce("[ureal_name] has designated the [name_type] as [html_decode(station_name())]", "Captain's Banner") //decode station_name to avoid minor_announce double encode
+	minor_announce("[ureal_name] [name_type] ismini değiştirerek [html_decode(station_name())] olarak belirledi.", "Kaptanın Sancağı") //decode station_name to avoid minor_announce double encode
```

Comment:
```suggestion

	var/locale_name_type = name_type
	switch(name_type)
		if("station")
			locale_name_type = "istasyonun"
		if("planet")
			locale_name_type = "gezegenin"

	minor_announce("[ureal_name] [name_type] adını [html_decode(station_name())] olarak belirledi.", "Kaptanın Sancağı") //decode station_name to avoid minor_announce double encode
```